packages/circuit-ui/components/Table/types.ts

```diff
@@ -19,12 +19,14 @@ export type SortByValue = boolean | number | string | Date;
 
 export type Direction = 'ascending' | 'descending';
 
-type SortableCellProps =
+export type CellAlignment = 'left' | 'right' | 'center';
+
+type SortableHeaderCell =
   | { sortable?: false; sortLabel?: never; sortByValue?: never }
   | {
       /**
-       * Makes a table column sortable when passed to a header cell. An
-       * accessible sortLabel also needs to be provided.
+       * Makes a table column sortable. An accessible sortLabel also needs to
```

Comment:
```suggestion
       * Makes a table column sortable. An accessible `sortLabel` also needs to
```

Reply:
I was _sure_ you'd comment on this when I saw it in the GitHub diff 😛

Reply:
Done in 9f74fe5